Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "yaml-adapter"
+version = "0.1.0"
+authors = ["closetool <c299999999@qq.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+yaml-rust = "0.4.5"
+async-trait = "0.1.42"
+casbin = "2.0"
+linked-hash-map = "0.5"
+thiserror = "1.0"
+async-std = { version = "1.8", features = ["attributes"] }
```

Comment:
Try to use `async-std   = { version = "1.8.0", default-features = false, optional = true }` to avoid runtime conflicts(as well as tokio).